addons/purchase_requisition/tests/common.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+from odoo.tests import common
+
+
+class TestPurchaseRequisitionCommon(common.TransactionCase):
+
+    def setUp(self):
+        super(TestPurchaseRequisitionCommon, self).setUp()
+
+        self.product_09_id = self.ref('product.product_product_9')
+        self.product_09_uom_id = self.ref('uom.product_uom_unit')
+        self.product_13_id = self.ref('product.product_product_13')
+        self.res_partner_1_id = self.ref('base.res_partner_1')
+        self.res_company_id = self.ref('base.main_company')
+        self.env.user.company_id.currency_id = self.env.ref("base.USD").id
+
+        self.ResUser = self.env['res.users']
```

Comment:
Avoid this. Don't put it on the class, and create a simple variable. 
Also, add the context key (i  don't remember the exact name) to avoid sending email. It should be like `no_password_reset` or something like this.